### PR TITLE
Add OpenShift Egress Router content

### PR DESCRIPTION
**Pull request by ahardin-rh**
Per https://github.com/openshift/openshift-docs/issues/2760, this pulls in content from https://github.com/openshift/origin/tree/master/images/router/egress

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL

File: admin_guide/managing_pods.adoc [hunk 1]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
 ====
 endif::openshift-enterprise,openshift-origin[]
 
+[[admin-guide-controlling-egress-traffic]]
+== Controlling Egress Traffic
+As an {product-title} cluster administrator, you can control egress traffic
+using either the xref:admin-guide-deploying-an-egress-router-service[router] or
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]: s/the/a/

File: admin_guide/managing_pods.adoc [hunk 2]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
...
+xref:admin-guide-limit-pod-access-egress[firewall] methods.
+
+[[admin-guide-limit-pod-access-egress-router]]
+=== Limiting Pod Access with an Egress Router
+
+The {product-title} egress router runs a service that redirects traffic to a
+specified remote server, using a private source IP address that is not used for
+anything else. The service allow pods to talk to servers that are set up
+to only allow access from white-listed IP addresses.
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]: s/white-listed/whitelisted/

File: admin_guide/managing_pods.adoc [hunk 3]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
...
+this pod.
+<2> Same value as the default gateway used by the node itself.
+<3>  Connections to the pod will be redirected to 203.0.113.25, with a source IP
+address of 192.168.12.99
+<4> The pod will only be deployed to nodes with the label site *springfield-1*.
+====
+
+The `pod.network.openshift.io/assign-macvlan annotation` creates a Macvlan
+network interface on the primary network interface, and then moves it into the
+pod's network name space before starting the egress-router container.
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]: Mark up egress-router with asterisks, maybe.

File: admin_guide/managing_pods.adoc [hunk 4]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
...
+
+The pod contains a single container, using the *openshift/origin-egress-router*
+image, and that container is run privileged so that it can configure the Macvlan
+interface and set up iptables rules.
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]:
```
`iptables`
```

File: admin_guide/managing_pods.adoc [hunk 5]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
...
+
+The environment variables tell the egress-router image what addresses to use; it
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]: Markup egress-router.

File: admin_guide/managing_pods.adoc [hunk 6]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
...
+will configure the Macvlan interface to use `*EGRESS_SOURCE*` as its IP address,
+with `*EGRESS_GATEWAY*` as its gateway.
+
+NAT rules are set up so that connections to any TCP or UDP port on the
+pod's cluster IP address will be redirected to the same port on
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]: s/will be/are/

File: admin_guide/managing_pods.adoc [hunk 7]
```diff
@@ -145,8 +145,108 @@ Note that the value of the override must be specified in string form.
...
+    port: 80
+  - name: https
+    port: 443
+  type: ClusterIP
+  selector:
+    name: egress-1
+----
+====
+
+Your pods can now connect to this service. Their connections will be
```
Comment (adellape) [inline admin_guide/managing_pods.adoc]: s/will be/are/

**Comment by adellape** [comment]
cc @danwinship This is handling https://github.com/openshift/openshift-docs/issues/2760.

**Comment by ahardin-rh** [comment]
@adellape Thanks! All updated.

**Pull request merged**